agml/synthetic/manual.py

```diff
@@ -196,9 +207,15 @@ def generate_manual_data(code = None,
 
     # Save the existing files for comparison in future runs.
     with open(os.path.join(SUPER_BASE_DIR, '.last_manual_compilation_cpp.cpp'), 'w') as f:
-        f.write(code)
+        if code is not None and not os.path.exists(code):
+            f.write(code)
+        else:
+            f.write(current_cpp)
     with open(os.path.join(SUPER_BASE_DIR, '.last_manual_compilation_cmake.txt'), 'w') as f:
-        f.write(cmake)
+        if cmake is not None and not os.path.exists(cmake):
+            f.write(cmake)
+        else:
+            f.write(current_cmake)
```

Comment:
The previous implementation did not work as intended.  Instead of copying the contents of the C++ code and CMake file to `'.last_manual_compilation_cpp.cpp` and `.last_manual_compilation_cmake.txt`, respectively, it would instead simply copy the paths. This change fixes that issue.